lib/fast_jsonapi/serialization_core.rb

```diff
@@ -57,7 +57,8 @@ def relationships_hash(record, relationships = nil, fieldset = nil, includes_lis
         relationships = {} if fieldset == []
 
         relationships.each_with_object({}) do |(key, relationship), hash|
-          included = includes_list.present? && includes_list.include?(key)
+          embedded_key = "#{record.class.name.underscore.pluralize}.#{key}".to_sym
```

Comment:
To me, this looks a little brittle in that it makes a lot of assumptions about how the key is defined, most notably that (a) the name of the property must also be the name of the class, and (b) the property must be plural.  It would also have a negative impact on performance because now it is doing that extra string manipulation work.

To me, the less guesswork the library does to determine these kinds of things, the better.

Lastly, I'm still not quite sure if the premise is valid, that if someone specifies `lazy_load_data: true` without `links`, it should behave this way.  The documentation about `lazy_load_data` is fairly sparse, but it is only mentioned within the context of `links`, which to me makes it reasonably clear that it should only be used with `links`.